docs/_integration-with-existing-apps-kotlin.md

```diff
@@ -110,11 +110,10 @@ This makes sure the React Native Gradle Plugin is available inside your project.
 Finally, add those lines inside your app's `build.gradle` file (it's a different `build.gradle` file inside your app folder):
 
 ```diff
-apply plugin: "com.android.application"
-+apply plugin: "com.facebook.react"
-
-repositories {
-    mavenCentral()
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
++   id("com.facebook.react")
 }
```

Comment:
Although using the `plugins{}` block is reccomended by Gradle, in this page we want to suggest a setup that is as close as possible the the existing template so by using the `apply:` syntax